src/pages/HackShackAttack/index.js

```diff
@@ -37,43 +39,138 @@ class Game extends Phaser.Scene {
   }
 }
 
+const gameConfig = {
+  initialize: true,
+  game: {
+    width: 1366,
+    height: 768,
+    parent: 'phaser-game',
+    type: Phaser.AUTO,
+    input: {
+      gamepad: true,
+      queue: true,
+    },
+    scale: {
+      autoCenter: Phaser.Scale.CENTER_BOTH,
+      mode: Phaser.Scale.ENVELOP,
+    },
+    pixelArt: true,
+    physics: {
+      default: 'arcade',
+      arcade: {
+        debug: false,
+        gravity: { y: 0 },
+      },
+    },
+    scene: Game,
+  },
+};
+
 const GameContainer = styled(Box)`
   position: relative;
   min-height: 1100px;
 `;
+
 const BackgroundWrapper = styled(Box)`
   position: absolute;
   z-index: 10;
 `;
 
-const HackShackAttack = () => {
-  const gameConfig = {
-    initialize: true,
-    game: {
-      width: 1366,
-      height: 768,
-      parent: 'phaser-game',
-      type: Phaser.AUTO,
-      input: {
-        gamepad: true,
-        queue: true,
-      },
-      scale: {
-        autoCenter: Phaser.Scale.CENTER_BOTH,
-        mode: Phaser.Scale.ENVELOP,
-      },
-      pixelArt: true,
-      physics: {
-        default: 'arcade',
-        arcade: {
-          debug: false,
-          gravity: { y: 0 },
-        },
-      },
-      scene: Game,
-    },
+const preventScroll = e => {
+  if ([32, 37, 38, 39, 40].indexOf(e.keyCode) > -1) {
+    e.preventDefault();
+  }
+};
+
+const TermsLayer = ({ setAccepted }) => {
+  const [checked, setChecked] = useState(false);
+  const [error, setError] = useState(false);
+
+  const handleSubmit = () => {
+    if (checked) {
+      setAccepted(true);
+      setError(false);
+      localStorage.setItem('formData', JSON.stringify({ accepted: true }));
+    } else {
+      setError(true);
+    }
   };
+
+  return (
+    <Layer>
+      <Box alignSelf="end" pad="small">
+        <Link to={{ pathname: '/arcade' }}>
+          <Button icon={<FormPreviousLink />} label="Back to Arcade" />
```

Comment:
I think just having the exit button ? Since we already have a back to arcade

Reply:
in order for the game to start they must accept. i dont think closing there layer here is an option. going back to arcade seems, like the best choice.

Reply:
got it makes sense!